src/components/Jobs/AllJobs/hook.ts

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable camelcase */
+import { useQuery } from '@tanstack/react-query';
+import config from 'config';
+import { handleApiErrors } from 'utils/handleApiErrors';
+import { handleError } from 'utils/handleUnAuthorization';
+
+const useFetchJobs = (organisationId: string | null) => {
+  const { data } = useQuery(
+    ['all_jobs'],
+    async () => {
+      let url;
+      if (organisationId) {
+        url = `${config.ApiBaseUrl}/job/all?organisation=${organisationId}`;
+      } else {
+        url = `${config.ApiBaseUrl}/job/all`;
+      }
+      const response = await fetch(url, {
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      });
+      const json = await handleApiErrors(response);
+      return json;
+    },
+    {
+      staleTime: Infinity,
```

Comment:
So that, it doesn't become stale after some time and refetch
But can try to remove it and see how things work.
For now, let's keep it infinity since, we won't get +jobs in a matter of minutes.